### DISCUSSION TITLE
2FA/TOTP for hosted Supabase admin panel?

### DISCUSSION THREAD

**Discussion by quicksnap** [supabase/supabase#1551] [Feature Requests]
Loving the hosted product so far! It's really helping me kick off an idea very fast! :)

If my idea magically springs into a more serious effort, having 2FA on the Supabase admin panel would be prudent. Wouldn't want any baddies coming in and droppin' my tables!

**Comment by quicksnap** [comment]
I just checked, and 2FA may be moot? If only OAuth is allowed for Supabase admin, the 2FA aspect is delegated to those providers. And, since the only provider is GitHub, and it supports 2FA, all is well.

**Comment by maggie44** [comment]
It seems the admin panel does now allow logins with an email address. But I can't see how to enable 2fa on my hosted admin panel? I am just being blind? Seems Supabase as a product supports 2fa, but the admin panel for setting up Supabase doesn't?

**Comment by tjmcewan** [comment]
https://github.com/orgs/supabase/discussions/12761
